src/Controls/src/Core/Platform/Android/Extensions/TextViewExtensions.cs

```diff
@@ -31,14 +32,21 @@ public static void UpdateLineBreakMode(this TextView textView, Label label)
 
 		public static void UpdateMaxLines(this TextView textView, Label label)
 		{
-			// Linebreak mode also handles settng MaxLines
+			// Linebreak mode also handles setting MaxLines
 			textView.SetLineBreakMode(label.LineBreakMode, label.MaxLines);
 		}
 
 		internal static void SetLineBreakMode(this TextView textView, LineBreakMode lineBreakMode, int? maxLines = null)
 		{
 			if (!maxLines.HasValue || maxLines <= 0)
-				maxLines = int.MaxValue;
+			{
+				// Without setting the MaxLines property, to equalize behaviors across platforms
+				// we set the max to 1.
+				if (lineBreakMode == LineBreakMode.TailTruncation)
+					maxLines = 1;
+				else
+					maxLines = int.MaxValue;
+			}
```

Comment:
Maybe we should leave this as is and move this _after_ the switch. We are setting 1 in many places so it feels better to rather isolates the 1 to the switches. And then we cleanup after the fact.

Reply:
At the setting, we could do:

```cs
textView.SetMaxLines(maxLines ?? int.MaxValue);
```